backend/models/helper/model.js

```diff
@@ -608,7 +608,26 @@ function downloadLatest(account, model) {
 	});
 }
 
-function uploadFile(req) {
+const checkTag = async (account, model, tag) => {
```

Comment:
Can we move this to history and rename this to isvalidTag() or something?

Since everything is doing is related to history it seems better fit that way